src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person =>
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  );
+  const age = men.reduce((sum, item) => sum + (item.died - item.born), 0);
```

Comment:
move this statement into separated function and reuse it throughout your program
```suggestion
  const age = men.reduce((sum, ({ died, born })) => sum + (died - born), 0);
```